ddpui/core/dashboard_chat/runtime.py

```
@@ -0,0 +1,960 @@
+                    user_query=state["user_query"],
+                    dashboard_summary=state["dashboard_summary"],
+                    retrieved_documents=state.get("retrieved_documents", []),
+                    sql=sql_validation.sanitized_sql if sql_validation else None,
+                    sql_results=state.get("sql_results"),
+                    warnings=warnings,
+                    related_dashboard_titles=[
+                        related_dashboard.title
+                        for related_dashboard in state.get("related_dashboards", [])
+                    ],
+                )
+            except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)